mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -418,9 +417,8 @@ func resourceMongoDBAtlasClusterCreate(ctx context.Context, d *schema.ResourceDa
 			return diag.FromErr(fmt.Errorf("`auto_scaling_disk_gb_enabled` cannot be true when provider name is TENANT"))
 		}
 
-		autoScaling = &matlas.AutoScaling{
-			DiskGBEnabled: pointy.Bool(false),
-		}
+		autoScaling = nil
```

Comment:
So I guess this is an issue of the JSON decoder setting the attribute as nil because is false? Does setting this to nil actually works?

Reply:
NVM autoscaling is an object

Reply:
yup it works because in the cluster property for AutoScaling is a pointer with the `omitempty` enabled, so the API doesn't need to declare this property in order to create a cluster